### PR TITLE
Add bl package to resolutions to fix vulnerability

### LINKED ISSUE #1437: Update bl indirect dependency
GitHub has reminded us of a security advisory in one of our dependencies.

### GitHub

> [**WS-2018-0074**](https://github.com/rvagg/bl/pull/22)
>
> **Vulnerable versions:** `< 0.9.5`
> **Patched version:** `0.9.5`
>
> Versions of bl before 0.9.5 and 1.0.1 are vulnerable to memory exposure.
>
> bl.append(number) in the affected bl versions passes a number to Buffer constructor, appending a chunk of uninitialized memory

### Yarn

`yarn audit` shows:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ moderate      │ Memory Exposure                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ bl                                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=0.9.5 <1.0.0 || >=1.0.1                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ react-native-level-fs                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ react-native-level-fs > levelup > bl                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/596                         │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

### PR DESCRIPTION
**Pull request by rekmarks**
Fixes #1437

See issue for details.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +59,8 @@
     ]
   },
   "resolutions": {
-    "**/optimist/minimist": "^1.2.5"
+    "**/optimist/minimist": "^1.2.5",
+    "react-native-level-fs/**/bl": "^0.9.5"
```
Comment (rekmarks) [inline package.json]: Is there any reason to not just do `**/bl` here?

Comment (kumavis) [reply package.json]: maybe if someone is using bl@1 somewhere

Comment (rekmarks) [reply package.json]: Ah, currently it's only `react-native-level-fs`, but the semver `^` will force `1.x` down to `0.x` if we add something else, right?

Comment (whymarrh) [reply package.json]: Yes, also the more targeted the resolution the better. This should be fine.

**Review by whymarrh — APPROVED** [review]
This seems legit to me—I'm not testing it functionally, but this diff makes sense

**Review by estebanmino — APPROVED** [review]
@rekmarks let's merge this

**Pull request merged**